src/utils/constants/User.ts

```diff
@@ -0,0 +1,30 @@
+import { IUser } from 'types';
+import { Language, Themes } from 'utils/enums';
+
+export const User = {
+	firebaseKey: 'users',
+	initInfo: {
+		id: '',
+		username: '',
+		email: '',
+		firstName: '',
+		lastName: '',
+		profilePicture: null,
+		games: [
```

Comment:
delete mock data if it is no longer needed

Reply:
Header, line 56, i use it as default info about user
if i need to delete it - should i try Redux or leave it like that?